src/terrasoApi/account/accountService.ts

```diff
@@ -55,20 +52,12 @@ export const getAuthURLs = () =>
     })
   );
 
-type UserQuery = {
-  users: {
-    edges: {
-      node: UserFragment & UserPreferencesFragment;
-    }[];
-  };
-};
-
 export const fetchProfile = async (
-  params: any,
+  params: null,
   currentUser: { email: string } | null
 ) => {
-  const query = `
-    query user($email: String!){
+  const query = graphql(`
```

Comment:
Question: how does the `graphql` function knows what fragments to add?

Reply:
at code generation time, the argument strings to the `graphql` function become keys in a dictionary where the values are the query with the fragments included (the fragments are also parsed at code generation time, must be tagged with `/* GraphQL */`. see https://github.com/techmatters/terraso-web-client/pull/934#discussion_r1195464222